davinci/models_save_flow_variable.go

```diff
@@ -0,0 +1,62 @@
+package davinci
+
+import "encoding/json"
+
+type _SaveFlowVariable SaveFlowVariable
+type SaveFlowVariable struct {
+	AdditionalProperties map[string]interface{} `json:"-" davinci:"-,unmappedproperties"` // used to capture all other properties that are not explicitly defined in the model
+	Key                  *float64               `json:"key,omitempty" davinci:"key,config,omitempty"`
+	Label                *string                `json:"label,omitempty" davinci:"label,config,omitempty"`
+	Name                 string                 `json:"name" davinci:"name,config"`
+	Type                 string                 `json:"type" davinci:"type,config"`
+	Value                interface{}            `json:"value,omitempty" davinci:"value,config,omitempty"`
```

Comment:
DaVinci tag should be one of the following:
- `davinci:"value,flowvariables,omitempty"` if flow value mappings are not exported
- `davinci:"value,config,omitempty"` if flow value mappings should be exported